db/c_level_db_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
-	cmn "github.com/tendermint/tm-cmn/libs/common"
+	cmn "github.com/tendermint/tm-cmn/common"
```

Comment:
this got changed I think. Sorry, we're in a process of separating some libraries.